source/manual/uptime-metrics.html.md

```diff
@@ -0,0 +1,36 @@
+---
+owner_slack: "#content-api"
+title: Uptime Metrics
+section: Monitoring
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2017-08-25
+review_in: 6 months
+related_applications: [publishing-api, content-store]
+---
+
+Uptime metrics are collected for the `content-store` and the `publishing-api`,
+they are available as [a Grafana dashboard][grafana-dashboard].
+
+![Uptime Metrics](images/uptime-metrics.png)
```

Comment:
Can we do any power moves here we write html that embeds the graph? Might be pure wishful thinking on my part.

Reply:
Yes, some experimentation looks like we can do this. But I think we have to wait until it's deployed to get the URL as I'm not sure which panel and dashboard ID it will be.